spec/features/step_by_step_navigation_spec.rb

```diff
@@ -20,16 +21,97 @@
       then_I_visit_the_navigation_steps_page_again
       and_I_see_my_selected_preferences
     end
+
```

Comment:
It's good that you've added feature tests 👍 but can you move them to where the controller actions were added.

It makes things easier to follow if the code change and the associated tests are in the same place.